lib/Agrammon/UI/CommandLine.pm6

```diff
@@ -28,14 +28,15 @@ my %*SUB-MAIN-OPTS =
   :named-anywhere,    # allow named variables at any location
 ;
 
-subset ExistingFile of Str where { .IO.e or $_ eq '-' or note("No such file $_") && exit 1 }
+subset OptionalExistingFile of Str where { !.defined or .IO.e or note("No such file $_") && exit 1 }
+subset ExistingFile of Str where { $_ ~~ Any or .IO.e or $_ eq '-' or note("No such file $_") && exit 1 }
```

Comment:
I think it wants to be changed here too, or if it's not optional then the `$_ ~~ Any` just removing?